components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -302,6 +306,14 @@ - (void)animationDidStop:(CAAnimation *)animation finished:(BOOL)flag {
 
 #pragma mark - Setters
 
+- (void)setElevation:(MDCShadowElevation)elevation {
+  if (elevation == _elevation) {
```

Comment:
Please use `MDCCGFloatEqual()`. Two floats that are very close to one another (but not exactly the same) wouldn't warrant invoking the block below.